src/Machine.Specifications.Runner.VisualStudio/Execution/VSProxyAssemblySpecificationRunListener.cs

```diff
@@ -15,6 +15,13 @@ public class VSProxyAssemblySpecificationRunListener :
 #endif
                                                             ISpecificationRunListener
     {
+#if !NETSTANDARD
+        [System.Security.SecurityCritical]
```

Comment:
Add to usings

Reply:
Consistent with other SecurityCritical attributes